### DISCUSSION TITLE
ÁFA arányosítás

### DISCUSSION THREAD

**Discussion by Szildiko** [nav-gov-hu/eVAT#27] [Kérdések és válaszok - Gépi interfész] [answered]
Tisztelt Illetékesek!

A programunkban folyamatban van a meglévő áfa arányosítás működésének az újraírása. Viszont, ha már lesz eÁFA, akkor arra gondoltunk, hogy úgy csinálnánk meg, hogy azzal kompatibilis legyen. (Ne kelljen kétszer dolgozni vele.)
A dokumentációban az áfaanalitika szerkezeténél tételsor adatként van feltüntetve a következő:
Adóelszámolási információk:
- Adókód
- Adókulcs
- Fizetendő: adóalap, adóösszeg
- Levonható: adóalap, adóösszeg
- Adómentesség oka
- Levonási hányad: levonási hányados, levonható adó összege

Ezt úgy kell értelmezni, hogy ha meg van az arányszám, és van 3 db költségszámla, amit meg kell osztani, akkor számlánként ki kell számolni az arányszámmal, hogy mennyi a levonható adó összege? 

Működő eljárás lehet-e, hogy a 3 számla adóösszegét összeadjuk és egy vegyes bizonylatban kerül rögzítésre, hogy abból mennyi, amit **nem** vonhat le? Ilyenkor az analitikára a 068-as sorra kerülne egy mínusz adó összeg. Természetesen a 3 számla adata is jelentésre kerülne alap és adóösszeg terén, csak levonható adó összege és a levonási hányados nem lenne tételesen kimutatva.

Tegyük fel, hogy az említett 3 számla januári volt az arányszám 0,77. Számokkal, hogy véletlenül se menjen félre a beszélgetés.
SZ01 alap 100.000 áfa 27.000 levonható 20.790 ft
SZ02 alap 37.037 áfa 10.000 levonható 7.700 ft
SZ03 alap 166.667 áfa 45.000 levonható 34.650 ft

Februárban módosul az arányszámom 0,89 tehát többet vonhatok le és lesz egy 4. számlám is. Akkor hogyan kellene a februári áfa analitikának kinéznie? Megint jelenteni kell a 3 számlát új levonható adó összeggel? Vagy csak a január és a február közti különbözettel? Értelemszerűen a 4. számlán nem intézhetem el, mert az közel sincs annyi összegben, valamint logika alapján ott is csak a 0,89-es arányszámmal kiszámolt összeget jelenthetném. Mi a helyzet az adóalappal?
SZ04 alap 18.519 áfa 5.000 ft levonható 4.450 ft
SZ01 új levonható 24.030 ft vagy a különbözet 3.240 ft
stb.

Ha ezt így kell értelmezni, akkor decemberre egy nagyon nagy mennyiségű 068-as analitika lesz. 
Arról még nem is beszéltem, hogyha az arányszám csökkenhet is, akkor meg kevesebbet kell levonni. 
Ha a vegyes bizonylat jelentése járható út, akkor viszont az aktuális időszaki számlák mellett lenne egy korrekciós bizonylat. A vegyes bizonylat használata se probléma mentes, mert az adóalap meghatározása ott is okozhat bonyodalmakat.

Előre is köszönök minden információt!

**Comment by NTCA-HM** [comment, marked as answer]
Kedves @Szildiko !

A deductionOptions csomópontot akkor kell kitölteni, amennyiben a bizonylat összege csak részben kerülhet levonásra. A deductionAmount minden esetben kitöltésre kell kerülnie. A deductionRate akkor kerül kitöltésre, amennyiben ez értelmezhető. Fontos figyelni ugyanakkor arra, hogy adóalapként (taxBase) a teljes összeg feltüntetésre kerül.
Amennyiben például egy bizonylat adóalapja 1000, adóösszege 270 Ft és 30%-os levonási korlátozás alá esik, akkor a következő módon szükséges a taxposition csomóponton belül a mezőket kitölteni:
•	TaxBase=1000
•	Taxamount=270
•	deductionRate=0,7
•	deductionAmount=189
Előfordulhat, hogy egy bizonylaton van olyan tételsor, amely tételes levonási tilalom alá tartozik. Ekkor nem lesz levonási hányados, kizárólag a deductionAmount kerül kitöltésre a deductionOptions csomóponton belül. 

A levonási hányadost és a levonható adó alapját, illetve összegét bizonylatonként kell meghatározni.
